src/custom_error/customError.ts

```diff
@@ -0,0 +1,16 @@
+export default class ValidationError extends Error {
+  private status: number;
+  constructor(message: string) {
+    super(message);
+    Error.captureStackTrace(this, this.constructor);
+    this.status = 404;
```

Comment:
You should also provide the status through the constructor. The status for the validation errors should be 400 not 404